### DISCUSSION TITLE
Wie mache ich ein Image Backup eienr Fritzbox? (6590 dual nand)

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#362] [Allgemein]
Hallo erst mal.

Ich hab da mal eine frage, wie kann ich bei einer 6590 (freetz-ng) ein backup der kommpletten FW machen?

**Comment by fda77** [comment]
`cat /dev/mtd* > datei` da kommt aber kein Image heraus sondern die einzelnen Partitionen. Ich glaube in der FAQ oder Wiki allgemein steht dazu was

**Reply by Conan179** [reply]
Ist das faq/wiki in überhaupt noch aktuell, in dem Thema?

**Reply by fda77** [reply]
Was gibt es den im Thema neues?

**Reply by Conan179** [reply]
@fda77 da muss ich erlich sagen, ich hab keine Ahnung, ich bin in der Technik der boxen nicht bewandert und kann nicht sagen, ob es noch aktuell ist. Das einzige was ich weiß ist, daß meine 7272,6590 und die 6591 2 Bands haben, meine alten beiden 7390 nicht. Deswegen meine Nachfrage, ob es noch aktuell ist.

**Reply by fda77** [reply]
Probier es doch einfach mal aus. Falls es Fehler gibt wirst du es sehen
